parm/aero/prep/prep_viirs.yaml.j2

```diff
@@ -0,0 +1,15 @@
+provider: VIIRSAOD
+window begin: {{ window_begin }}
```

Comment:
do we need ` | to_isotime ` here on these two lines?
`{{ window_begin | to_isotime }}`

Reply:
What's the purpose of` | to_isotime`? Now it works fine without it, but I can add it if needed.

Reply:
I think it should ensure that the date is formatted `YYYY-MM-DDTHH:MM:SSZ`